scripts/run_validator.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
-tag=latest
+tag=add-dynamic-options-identifier-check
```

Comment:
Reminder to revert before merge.
```suggestion
tag=latest
```